<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myWidget</class>
 <widget class="QWidget" name="myWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>服务器端口</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditPort">
     <property name="text">
      <string>27783</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>服务器ip</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditip">
     <property name="text">
      <string>103.46.128.45</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QTextEdit" name="textEditRead">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QTextEdit" name="textEditWrite"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="ButtonSend">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>575</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="ButtonClose">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="ButtonConnect">
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
